geonode/base/api/views.py

```diff
@@ -1076,7 +1076,16 @@ def resource_service_update(self, request, pk):
         url_name="resource-service-copy",
         methods=["put"],
         permission_classes=[
-            IsAuthenticated, UserHasPerms
+            IsAuthenticated, UserHasPerms(
```

Comment:
@mattiagiupponi could you also add documents since it also has the `download_resource` perm.